public/manifest.json

```
@@ -0,0 +1,24 @@
+{
+  "manifest_version": 3,
+  "name": "Meli a Crypto",
+  "description": "Cambia los precios de Mercadolibre a Crypto en tiempo real",
+  "version": "1.0",
+  "options_ui": {
+    "page": "options.html"
+  },
+  "action": {
+    "default_icon": "icon.png",
+    "default_popup": "popup.html"
+  },
+  "content_scripts": [
+    {
+      "matches": ["<all_urls>"],
```

Acá no estaría bueno que matchee solo el dominio de Meli? Para evitar que se ejecute en cualquier lado?
```suggestion
      "matches": ["https://www.mercadolibre.com.ar/*"],
```

---

Listo 👌